webpack.config.js

```diff
@@ -255,7 +254,8 @@ module.exports = {
     },
     {
       test: /\.svg$/,
-      loaders: ['babel-loader', 'react-svg-loader'],
+      // loaders: ['babel-loader', 'react-svg-loader'], // to address the `css-what` vulnerability issue, after updating to webpack 5 and latest `react-svg-loader` we can switch back to this
```

Comment:
we might need the same comment to explain [here](https://github.com/uc-cdis/data-portal/blob/f20b3d2c7fca4cf8e5b4b3adfb61cc14d1a26bbf/.storybook/webpack.config.js#L39)?

Reply:
nice catch, added that